django/core/checks/security/base.py

```diff
@@ -175,6 +178,20 @@ def check_secret_key(app_configs, **kwargs):
 
 
 @register(Tags.security, deploy=True)
+def check_secret_key_valid_unicode(app_configs, **kwargs):
+    valid_unicode = True
+    try:
+        # To maintain backwards compatibility, instead of checking that
+        # SECRET_KEY is a valid unicode string, simply check if it goes through
+        # `force_bytes` and `force_text` without throwing an exception.
+        force_bytes(settings.SECRET_KEY)
+        force_text(settings.SECRET_KEY)
+    except UnicodeDecodeError:
```

Comment:
copying Baptiste's comment from the other PR: Shouldn't this be `except UnicodeError`. I think this points to the fact that more tests are needed, in particular I'd like to see one where `force_bytes(SECRET_KEY)` actually blows up.